### DISCUSSION TITLE
文档typo以及此项目的整体目标

### DISCUSSION THREAD

**Discussion by MillerTheBest** [SylarLong/iztro#167] [Q&A]
Hi，

1. 我刚刚试了一下https://ziwei.pro/quick-start.html#%E5%BC%95%E5%85%A5%E4%BB%A3%E7%A0%81, html tab里的例子asto是个笔误，应该是astro。
2. 请问这个项目的最终目的是什么？ 做到一个完备的开源的排盘，还是未来会往更多的开发走？

谢谢

**Comment by SylarLong** [comment]
@MillerTheBest 
1. 感谢指正，确实有个笔误。
2. 目前没有什么最终目的，边走边看。

感谢关注 ❤️

**Reply by MillerTheBest** [reply]
谢谢 :)
